Bakker/models/bakker_verkoop.py

```diff
@@ -0,0 +1,328 @@
+from odoo import models, fields, api
+from odoo.exceptions import ValidationError
+import base64
+
+class BakkerVerkoop(models.Model):
+    _name = "bakker_verkoop"
+    _description = "Verkoop van bakkerij koeken"
+    _inherit = ['mail.thread', 'mail.activity.mixin'] 
+    _order = "verkoop_datum desc"
+    
+    
+    name = fields.Char(string="Verkoop Nummer", required=True, default="Nieuw", tracking=True)
+    koek_id = fields.Many2one('bakker_koeken', string="Koek", required=True, tracking=True)
+    partner_id = fields.Many2one('res.partner', string="Klant", required=True, tracking=True)
+    klant_email = fields.Char(string="Email", related='partner_id.email', readonly=True)
+    klant_telefoon = fields.Char(string="Telefoon", related='partner_id.phone', readonly=True)
+    aantal = fields.Integer(string="Aantal", required=True, default=1, tracking=True)
+    prijs_per_stuk = fields.Float(string="Prijs per stuk", required=True, tracking=True)
+    korting_percentage = fields.Float(string="Korting (%)", default=0.0, tracking=True)
```

Comment:
```suggestion
    korting_percentage = fields.Float(string="Korting (%)", tracking=True)
```
If I am not mistaken in odoo, a float field has a default value of 0.0